src/Components/Card.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+function Card(props) {
+  return (
```

Comment:
Very nice! Maybe you could destructure props here?🙂

Reply:
done